.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    container:
+      image: docker.io/fedora:39
+      options: --sysctl net.ipv6.conf.lo.disable_ipv6=0
```

Comment:
I don't know if nss tests fail with this. If so, I suggest removing.

Reply:
Its a bit counter-intuitive, but that line explicitly _enables_ ipv6, which I think is what you mentioned was required before, and so the nss and gnutls tests are passing currently.

Reply:
Ah, yes, sorry for the noise :man_facepalming: